packages/legend-art/style/base/_input.scss

```diff
@@ -34,6 +34,10 @@
   }
 }
 
+.input--small {
```

Comment:
I think this is something we put in `panel.scss` instead of here. This small input width is really just a panel thing

Reply:
true!